SmartDeviceLink/SDLSyncMsgVersion.h

```diff
@@ -24,7 +24,7 @@
 - (instancetype)initWithDictionary:(NSMutableDictionary *)dict;
 
 // TODO: (Alex M.)[2016-12-1] Change from NSInteger to UInt8
-- (instancetype)initWithMajorVersion:(NSInteger)majorVersion minorVersion:(NSInteger)minorVersion __deprecated_msg("Include patchVersion");
+- (instancetype)initWithMajorVersion:(NSInteger)majorVersion minorVersion:(NSInteger)minorVersion __deprecated_msg("Use (instancetype)initWithMajorVersion:(NSInteger)majorVersion minorVersion:(NSInteger)minorVersion patchVersion:(NSInteger)patchVersion instead");
```

Comment:
I should have been clearer, for consistency with other messages in the library, this should say:

> "Use initWithMajorVersion:minorVersion:patchVersion: instead"